app/Http/Controllers/UserController.php

```diff
@@ -197,4 +203,46 @@ public function editPassword(EditPasswordRequest $request)
             }
         }
     }
+
+    public function postFollow(Request $request, User $user)
+    {
+        $input = $request->all();
+        $currentUser = Auth::user();
+
+        if ($request->ajax() && !empty($input['target_id']) && !empty($input['target_type']) && Auth::check()) {
+            $input['user_id'] = Auth::id();
+            $result = '';
+
+            if ($input['target_type'] == 1) {
+                $result = $this->userRepository->saveFollow($input);
+            } elseif ($input['target_type'] == 2) {
+                $result = $this->tagRepository->saveFollow($input);
+            } else {
+                return $this->returnResponse(__('alert.error.follow'), 401);
+            }
+
+            if (!empty($result)) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 224</br>Blank line found at start of control structure